src/util/jszip.js

```diff
@@ -1,9155 +1,9412 @@
 /*!
```

Comment:
To be honest I had the same feeling, but I double checked, and no, proj4-src.js and jszip.js have stayed the same in the develop branch since 2 years ago. Maybe we're thinking about #155, which added the CommonJS interface to the whole library, alongside its previously present AMD one.

I did mention proj4's pure AMD interface in [task #157](https://github.com/NASAWorldWind/WebWorldWind/issues/157#issuecomment-315753325), but that issue changed a lot while I was learning the implications of the related technologies. At first I was suspecting React, then Webpack, and it ended being something related to the CommonJS interfacing itself, development toolchain notwithstanding. After switching for the pure AMD interface in proj4, the package even loads inside Node (of course it halts when the library tries to access the web browser global object).

I'll revert to our old versions, add the pure AMD interface to proj4-src.js, and test the package.